pyensign/events.py

```diff
@@ -96,6 +96,18 @@ def proto(self):
             created=self.created,
         )
 
+    def published(self):
+        """
+        Returns True if the event has been published (sent to the server).
+        """
+
+        print(self._state)
+        return (
+            self._state == EventState.PUBLISHED
+            or self._state == EventState.ACKED
+            or self._state == EventState.NACKED
+        )
```

Comment:
Might be easier to do `self._state >= EventState.PUBLISHED` -- in the case where the `_state == EventState.SUBSCRIBED` -- this also implies published and if they're in different queues it shouldn't matter to the overall implementation. `>=` is much better than a series of `OR`.